export const valueChainErrors = {
  duplicateName: {
    message: 'This name is already being used by another value chain',
    userMessage: 'Este nome já está sendo utilizado por outra cadeia de valor',
  },
  notFound: {
    message: 'value chain not found',
    userMessage: 'cadeia de valor não encontrada',
  },
  deleteWithTrackers: {
    message: 'value chains with trackers cannot be deleted',
    userMessage: 'cadeias de valor com trackers registrados não podem ser excluida',
  },
};
